org.codeberg.LibreMenuEditor.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.codeberg.LibreMenuEditor
```

Comment:
this app-id is not valid:

- code hosting always require 4 component. 3 being the user/org and 4 being the app
- it must use the domain for hosting webpage.

Reply:
so one of these would be ok?

org.codeberg.LibreMenuEditor.LibreMenuEditor
org.codeberg.libremenueditor.LibreMenuEditor
org.codeberg.libre-menu-editor.LibreMenuEditor

Reply:
neither.

`page.codeberg.libre_menu_editor.LibreMenuEditor`

Reply:
Why "page.codeberg.libre_menu_editor.LibreMenuEditor" and not "org.codeberg.libre_menu_editor.LibreMenuEditor"?

Reply:
Does it need to be "page".codeberg.libre_menu_editor.LibreMenuEditor? I would like it much better if it started with "org"

Reply:
Never mind, I think I misunderstood when you said "it must use the domain for hosting webpage". I will create a webpage with codeberg.page and update the app-id to page.codeberg.{username}.{app-name}

Reply:
you don't need to create anything here. Just to set the app-id as needed.

Reply:
My username on codeberg is "libre-menu-editor" and not "libre_menu_editor". So https://libre-menu-editor.codeberg.page works and https://libre_menu_editor.codeberg.page does not work. If someone created a codeberg-account with the username "libre_menu_editor", there could be confusion.

Therefore I assume that I should use the app-id "page.codeberg.libre-menu-editor.LibreMenuEditor" instead of "page.codeberg.libre_menu_editor.LibreMenuEditor". Please tell me which one is correct.

It would be nice if the all the names were as consistent as possible.